VERSION

```diff
@@ -0,0 +1 @@
+1.0.0-pre1
```

Comment:
Tracking the tilejson bit here: https://github.com/tilezen/vector-datasource/issues/962.